cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/RangeAnalysisStage.qll

```diff
@@ -110,7 +110,7 @@ signature module DeltaSig {
   Delta fromInt(int n);
 
   bindingset[f]
-  bindingset[result]
+  //bindingset[result]
```

Comment:
Should this commented-out annotation be deleted now?

Reply:
Yes, it should